src/QuLDE.jl

```diff
@@ -219,27 +300,45 @@ function DiffEqBase.solve(prob::QuLDEProblem{uType,tType,isinplace, F, P}, alg::
     return out
 end;
 
+function DiffEqBase.solve(prob::QuLDEProblem{uType, tType, isinplace, F, P, true}, alg::QuLDE; kwargs...) where {uType, tType, isinplace, F, P}
+    M = prob.A
+    b = prob.b
+    t = prob.tspan[2] - prob.tspan[1]
+    Mu = M/opnorm(M)
+    k = alg.k
+    siz, = size(M)
+    nbit = log2i(siz)
+    l = 2
+    blk = QuLDEProbParam(k,t,l,prob)
+    n = k*(1 + l) + nbit
+    CPType = get_param_type(blk)
+    inreg = ArrayReg(b/norm(b)) ⊗ zero_state(CPType, k*l) ⊗ zero_state(CPType, k)
+    cir = quldecircuit(blk,Mu,nbit,n)
+    r = apply!(inreg,cir) |> focus!(1:n - nbit...,) |> select!(0) |> state
```

Comment:
I'm wondering if we should provide `focus!(::UnitRange)` for down stream users? It seems not necessary to write in this way. What do your guys think? @dgan181  @GiggleLiu

Reply:
Yes. It sounds like a good idea to me.

Reply:
@dgan181 do you want to give a PR on this later?

Reply:
Yes, I could do that.